libs/api/domains/national-registry-x-road/src/lib/national-registry-x-road.module.ts

```diff
@@ -0,0 +1,42 @@
+import { DynamicModule, Module } from '@nestjs/common'
+import { NationalRegistryModule } from '@island.is/clients/national-registry-v2'
+import { createXRoadAPIPath, XRoadMemberClass } from '@island.is/utils/api'
+
+import { NationalRegistryXRoadResolver } from './national-registry-x-road.resolver'
+import { NationalRegistryXRoadService } from './national-registry-x-road.service'
```

Comment:
The filenames aren't following the file naming pattern for island.is.

It's correct to use `kebab-case` for library names, but the filenames + directories within should use `camel-case`. (`national-registry-x-road.resolver` => `nationalRegistryXRoad.resolver`).